cpu/qn908x/periph/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_PERIPH_FLEXCOMM
+    bool
+    depends on TEST_KCONFIG
+    default y if MODULE_PERIPH_UART || MODULE_PERIPH_I2C
```

Comment:
Shouldn't SPI also select this?
@iosabi

Reply:
very likely, SPI is a flexcomm.

Reply:
I can add it, it's not in the Makefile.dep

Reply:
maybe I forgot, thanks!

Reply:
Done